cdc/log.cc

```diff
@@ -737,7 +750,15 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
             // so this is premature.
             auto f = trans.pre_image_select(qs.get_client_state(), db::consistency_level::LOCAL_QUORUM, m);
             return f.then([trans = std::move(trans), &mutations, idx] (lw_shared_ptr<cql3::untyped_result_set> rs) mutable {
-                mutations.push_back(trans.transform(mutations[idx], rs.get()));
+                auto& m = mutations[idx];
+                auto s = m.schema();
```

Comment:
nitpick: this makes a copy of schema, incrementing and decrementing its reference counter. Do we need to?

Reply:
It will be copied anyway in each `split(m, s)` call, but OK, I changed this to `auto&`, one less copy.